docs/structured-streaming-kafka-integration.md

```diff
@@ -770,7 +770,9 @@ Delegation tokens can be obtained from multiple clusters and <code>${cluster}</c
     <td><code>spark.kafka.clusters.${cluster}.security.protocol</code></td>
     <td>SASL_SSL</td>
     <td>
-      Protocol used to communicate with brokers. For further details please see Kafka documentation. Only used to obtain delegation token.
+      Protocol used to communicate with brokers. For further details please see Kafka documentation. Protocol is applied on all the sources and sinks as default where
+      <code>bootstrap.servers</code> config matches (for further details please see <code>spark.kafka.clusters.${cluster}.target.bootstrap.servers.regex</code>)
+      , and can be overridden by setting <code>kafka.security.protocol</code> on the source or sink.
```

Comment:
Well, to be nitpicky, the comma goes on the previous line.

Reply:
I've had similar feeling when a guy suggested this in a previous state, fixed :)